packages/gestalt/src/Contents.js

```diff
@@ -371,7 +371,7 @@ export default class Contents extends React.Component<Props, State> {
     mainDir: null,
   };
 
-  flyout = React.createRef();
+  flyout: {| current: null | HTMLDivElement |} = React.createRef();
```

Comment:
agree we dont have a nice polymorphic type, but can we use `React.ElementRef` maybe?
https://github.com/facebook/flow/commit/d74ab13a8dfc5395ec5883aa2fa56be0a03a3a3f
https://flow.org/en/docs/react/types/#toc-react-elementref